bookkeeper-server/src/main/java/org/apache/bookkeeper/proto/PacketProcessorBase.java

```diff
@@ -119,12 +121,17 @@ protected void sendResponse(int rc, Object response, OpStatsLogger statsLogger)
         }
 
         if (channel.isActive()) {
-            channel.writeAndFlush(response, channel.voidPromise());
+            ChannelPromise promise = channel.newPromise().addListener(future -> {
+                if (!future.isSuccess()) {
+                    logger.debug("Netty channel write exception. ", future.cause());
```

Comment:
Why not log it as an error?

Reply:
When this error occurs, it means that the client has already close. Log error is unnecessary, the other log will show the connection already close.

Reply:
I think this need to print the error
`logger.error("Netty channel write exception. ", future.cause());`